### PR TITLE
Home work 3 done!

### PR DESCRIPTION
**Pull request by Everett-Young**

### CODE REVIEW

File: homeworks/pavel.krasovskiy_everett-young/homework_3/homework3.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var height = 4;
```
Comment (donVladimir) [inline homeworks/pavel.krasovskiy_everett-young/homework_3/homework3.js]: please use const instead of var

Comment (Everett-Young) [reply homeworks/pavel.krasovskiy_everett-young/homework_3/homework3.js]: Done! Thanks.

Comment (donVladimir) [reply homeworks/pavel.krasovskiy_everett-young/homework_3/homework3.js]: YAW, It seems, there are more constants in your code ;)

Comment (Everett-Young) [reply homeworks/pavel.krasovskiy_everett-young/homework_3/homework3.js]: fixed)

**Pull request merged**